Log in and Sign up pages

PetraMihucz opened the pull request:
HomePage - made the variable false so you can go to the two pages
index.js - added the routes for the two pages
LogIn and SigUp pages - added the input fields and linked to the rest of
the website

PetraMihucz committed:
Log in and Sign up pages

HomePage - made the variable false so you can go to the two pages
index.js - added the routes for the two pages
LogIn and SigUp pages - added the input fields and linked to the rest of
the website

ValerieBeltrame commented on festipals/src/components/LogInPage.js:
```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+
+export default class HomePage extends Component {
```
class LogInPage instead of HomePage

ValerieBeltrame commented on festipals/src/components/LogInPage.js:
```diff
@@ -0,0 +1,55 @@
+                            <h1>Log In</h1>
+                    </div>
+                  </div>
+              </div>
```
Could we use the PageHeader component here instead?

ValerieBeltrame commented on festipals/src/components/LogInPage.js:
```diff
@@ -0,0 +1,55 @@
+
+                                </div>
+                                </div>
+
```
Can you fix the indentation here please? Also remove some of the empty lines, just so it looks a bit nicer.

ValerieBeltrame commented on festipals/src/components/SignUpPage.js:
```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+
+export default class HomePage extends Component {
```
SignUpPage instead of HomePage

ValerieBeltrame commented on festipals/src/components/SignUpPage.js:
```diff
@@ -0,0 +1,67 @@
+                            <h1>Sign up</h1>
+                    </div>
+                  </div>
+              </div>
```
Same comment as on the login page about the PageHeader

ValerieBeltrame reviewed (commented):
Just some small details, the rest looks good to me :)

PetraMihucz committed:
Small mistakes changed

class renamed to LogInPage and SignUpPage
deleted extra space
added the PageHeader compnent
